src/data-mgt/python/jobs/device_registry.py

```diff
@@ -0,0 +1,131 @@
+import json
+import os
+import requests
+
+AIRQO_API_BASE_URL = os.getenv("AIRQO_API_BASE_URL")
+
+MEASUREMENT_UNITS = dict({
+    "temperature": "°C",
+    "relHumid": "%",
+    "pm10ConcMass": "μg/m3",
+    "pm10ConcNum": "micrograms",
+    "pm1ConcMass": "μg/m3",
+    "pm1ConcNum": "micrograms",
+    "pm2_5ConcMass": "μg/m3",
+    "pm2_5ConcNum": "micrograms",
+    "no2Conc": "ppb",
+    "vocConc": "ppb",
+    "co2Conc": "ppb",
+})
+
+
+def single_component_insertion(data, tenant):
+
+    """
+    inserts a single device component data into the events table
+    :param data: component data (includes component name and device id to reduce on function args)
+    :param tenant: organisation eg airqo
+    :return: none
+    """
+
+    try:
+
+        # extract the component name and device id from the data
+        device = data.pop("device")
+
+        # create a json object of the remaining data and post to events table
+        json_data = json.dumps([data])
+        # print(json_data)
+        headers = {'Content-Type': 'application/json'}
+        url = AIRQO_API_BASE_URL + "devices/events/add?device=" + device + "&tenant=" + tenant
+
+        results = requests.post(url, json_data, headers=headers)
+
+        print(results.json())
+
+    except Exception as e:
+        print("================ Error Occurred ==================")
+        print(e)
+        print("================ Error End ==================")
+
+
+def get_component_type(component, tenant):
```

Comment:
With the current flow of things (device focused), I think the component types and components are no longer necessary checks in our streaming works.